src/pybind/mgr/diskprediction/agent/__init__.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+
+
+from requests.auth import HTTPBasicAuth
+
+from ..common import timeout, TimeoutError
+from ..common.restapi import RestAPI, gen_configuration
+
+
+
+class Command(object):

```

Comment:
The naming a bit confusing here.  instances of this class aren't really commands: this class is an API client.

This is also the only user of the RestAPI class (which is defined in common/, although it isn't shared with anything else).

I'd suggest refactoring this so that you have one "RestApiClient" class instead of RestAPI+Command.